### ISSUE TITLE
Feature: Shellcheck disable all

### ISSUE THREAD

**Issue by andrewmiller1** [koalaman/shellcheck#1355]
Use case: When transitioning from pseudocode to actual code, you may want the linting disabled for parts of your code in a single file.

1. By dropping a `# shellcheck disable=all` below what you're currently working on, you can keep linting enabled, without being bombarded by violations you haven't even got to yet.
2. Pairing with a `# shellcheck enable=all` can allow you more flexibility in what parts of the file in progress you chose to work on and when.

This will keep the relevant errors that grab your attention to be the only ones that do.

**Comment by anishmittal2020** [comment]
This is also useful when we want to try out others script when they don't use shellcheck, and we can avoid resolving tons of issues.

**Comment by felipec** [comment]
Exactly, I'm editing somebody else's script, I don't want to see all red all over the place.
